Automattic/jetpack
# Add "allow download" option for VideoPress video

Fixes Automattic/greenhouse#1062

#### Changes proposed in this Pull Request:

This PR adds the "allow download" option for VideoPress videos, allowing the video to be displayed with a "Download video" button linking to the original video.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* On a sandboxed jetpack with jurassic tube and VideoPress enabled
* Sandbox WPCOM public-api.wordpress.com
* Apply the wpcom patch stack
    * phab co D72213
    * arc patch D72220  
* Go to your media library /wp-admin/upload.php
* Select or upload and select a VideoPress video to open the attachment details
* ✅ The following checkbox should be visible

![Capture d’écran 2021-12-28 à 15 04 07](https://user-images.githubusercontent.com/1420594/147574166-12fa4ed7-0639-453c-914a-59d8f98fc3b6.png)

* Toggle the checkbox and reload the page
* ✅ The checkbox should be in the same state when reloading the page 
* ✅ No other setting should be affected by this change
* Create or edit an existing post using the block editor
* Add a /video block
* Select the VideoPress video from your media library
* ✅ The "Allow download" toggle should be visible
 
![Capture d’écran 2021-12-28 à 15 10 09](https://user-images.githubusercontent.com/1420594/147574693-35c9c39d-9e31-486d-bf8c-e110412f9076.png)

* ✅ The "Allow download" toggle should be in the same state as in the media  library
* Toggle the "allow download"  setting, save and reload the page
* ✅ The "allow download" toggle should be in the same state
